### PR TITLE
guess conntrack netns ino independently of tracer netns ino

### PR DESCRIPTION
**Pull request by akarpz**
### What does this PR do?

Currently the `ct_netns_ino` offset value in conntrack offset guessing is set to the one that is guessed via tracer offset guessing since it should be the same offset. In reality this means that even when running the CO-RE tracer, tracer offset guessing is still required to support conntrack (which is always prebuilt). This PR removes that link between the two modules.

### Motivation

https://datadoghq.atlassian.net/browse/NPM-3143

### Reviewer's Checklist

- [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
- [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
- [ ] Changed code has automated tests for its functionality.
- [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
- [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
- [ ] If applicable, the [config template](https://github.com/DataDog/datadog-agent/blob/main/pkg/config/config_template.yaml) has been updated.

### CODE REVIEW

File: pkg/network/tracer/offsetguess/offsetguess.go [hunk 1]
```diff
@@ -123,6 +128,30 @@ func enableProbe(enabled map[probes.ProbeFuncName]struct{}, name probes.ProbeFun
 	enabled[name] = struct{}{}
 }
 
+func boolToUint64(in bool) uint64 {
```
Comment (hmahmood) [inline pkg/network/tracer/offsetguess/offsetguess.go:131]: There is already a helper somewhere for this I think.

File: pkg/network/tracer/offsetguess/tracer.go [hunk 2]
```diff
@@ -1090,76 +1091,18 @@ type tracerOffsets struct {
...
 	offsetBuf, err := netebpf.ReadOffsetBPFModule(cfg.BPFDir, cfg.BPFDebug)
 	if err != nil {
 		o.err = fmt.Errorf("could not read offset bpf module: %s", err)
 		return nil, o.err
 	}
 	defer offsetBuf.Close()
-
-	o.offsets, o.err = RunOffsetGuessing(cfg, offsetBuf, NewTracerOffsetGuesser)
-	return fromConfig(cfg, o.offsets), o.err
+	return RunOffsetGuessing(cfg, offsetBuf, NewTracerOffsetGuesser)
```
Comment (hmahmood) [inline pkg/network/tracer/offsetguess/tracer.go:1105]: We should still store the offsets in `o.offsets`

Comment (hmahmood) [reply pkg/network/tracer/offsetguess/tracer.go:1105]: And store the error `o.err`

File: pkg/network/tracer/offsetguess/offsetguess.go (hunk 1 above)
Comment (akarpz) [reply pkg/network/tracer/offsetguess/offsetguess.go:131]: the only thing I could find was `boolToBinary` in the usm package, but that returns `uint8`

**Review by hmahmood — APPROVED** [review]

**Pull request merged**